src/main/java/Compartor.java

```diff
@@ -0,0 +1,5 @@
+
+@FunctionalInterface
+public interface Compartor<T>{
+    int compare(T o1,T o2);
```

Comment:
les parameters de mieux ikounou déclarés `final` bach hakda katban l'intention de ne pas les modifier.

Reply:
oui , hadi kayna

Reply:
pour le nommage, ana banlia mieux left et right, hit dak l'ordre tinfa3 ila bghina nfehmou le résultat de la méthode

Reply:
7ta hada blan oui , je le modifie

Reply:
hhhh , just noticed it